quickfixj-base/src/main/java/quickfix/Dictionary.java

```diff
@@ -25,7 +25,7 @@
 /**
  * Name/value pairs used for specifying groups of settings.
  *
- * @see SessionSettings
+ * see SessionSettings
```

Comment:
Is there a reason why you removed that `@see` javadoc tag?

Reply:
The reference is to a class that depends on the Dictionary, Dictionary does not depend on SessionSettings, so it's the wrong way round. That class is in quickfixj-core so the @see tag causes Javadoc warnings or errors, core depends on base not visa versa. I did not want to be aggressive and lose the information that the author intended so I just took away the "@" so that no errors result.